content/posts/irods-development-update-may-2025.md

```diff
@@ -0,0 +1,297 @@
+Title: iRODS Development Update: May 2025
+Date: 2025-05-19 10:00
+Author: Kory Draughn
+Slug: irods-development-update-may-2025
+Status: published
+
+
+Hello reader,
+
+iRODS 5 is almost ready for prime time. As of this development update, all issues have been closed and we have entered the testing phase. Let's talk about what's changed since last month's update.
```

Comment:
```suggestion
iRODS 5 is almost ready for prime time. As of this development update, all issues have been closed and we have entered the confirmation testing phase. Let's talk about what's changed since last month's update.
```

Reply:
Done.